docs_dev/releasing.md

```diff
@@ -179,27 +186,51 @@ Alternatively you can use the UI.
    ```
    * Set the tag to be the correct version for the tag.
 
-## Version the website
+# Version the website
 
 The main Kubeflow website at [www.kubeflow.org](www.kubeflow.org) points to the
-**master** branch of the `kubeflow/website` repo. Similarly, 
-[master.kubeflow.org](https://master.kubeflow.org/) also points to the master
-branch.
+version of the website we want users to see.
+
+* Most of the time this will correspond to the **master** branch of the `kubeflow/website` repo
+
+* However, leading up to the release of vY.0.0 [www.kubeflow.org](www.kubeflow.org) 
+  will point at a branch containing a stable version of the docs corresponding to vX.0.0
 
 For each minor (X.Y) release, we also publish a corresponding version of the 
 website. Each version of the website is generated from a separate 
 [branch](https://github.com/kubeflow/website/branches)
-of the `kubeflow/website` repository. The naming convention is as follows:
+of the `kubeflow/website` repository. 
+
+* The naming convention is as follows:
 The site at `vX-Y.kubeflow.org` points to the release at vX.Y-branch.
 For example, the [documentation for v0.6](https://v0-6.kubeflow.org) is 
 maintained in the
 [v0.6-branch](https://github.com/kubeflow/website/tree/v0.6-branch).
 
-There are two release phases when we need to update the website:
-* Updating version numbers etc for an upcoming release (major, minor, or patch).
-* Creating a website branch for the latest major or minor release.
+* Per [kubeflow/website#1581](https://github.com/kubeflow/kubeflow/pull/4717) the naming
+  convention might change when we switch to using Netlify's deploy branch feature
```

Comment:
I removed it. I'm hoping nothing will have to change and we can keep the same convention. The only reason it would have to change is if we hit SSL issues because we would be using a different hostname "v0-8" then what Netlify uses . At which point the options would be

1. Use the branch name "v0-X-Y" to generate the hostnames we want
1. Switch to hosting archived docs at "v0-7-branch"

I'm hoping nothing will have to change or we can find an appropriate work around.

I'm prioritizing 1.0 though and kicking the can down the road.